package.json

```diff
@@ -50,6 +52,7 @@
     "sass-loader": "3.1.2",
     "striptags": "2.1.1",
     "style-loader": "0.13.0",
+    "url-loader": "0.5.7",
```

Comment:
I think the `url-loader` will serve the asset via base64 encoding in a url if it's small enough (which is why it's a little niftier than `file-loader`)